tox.ini

```diff
@@ -23,3 +23,13 @@ deps =
 commands =
   rm -rf releasenotes/build/html
   sphinx-build -n -W --keep-going -b html releasenotes/source releasenotes/build/html
+
```

Comment:
Could you add the docs env to the matrix in the github actions CI pull request workflow?

`.github/workflows/stackhpc-pull-request.yml`